front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -98,6 +99,31 @@ export function InstructionScreen({
   ) => void;
   setEdited: (edited: boolean) => void;
 }) {
+  const editor = useEditor({
+    extensions: [
+      StarterKit.configure({
+        heading: false,
```

Comment:
`StarterKit` bundles a lot of [options](https://tiptap.dev/docs/editor/api/extensions/starter-kit), should we instead only load the one we need?

Reply:
One Document, Paragraph and Text extensions are required for our use case I guess.

Reply:
So I must implement those myself ?